roles/ceph-mon/tasks/docker/copy_configs.yml

```diff
@@ -8,7 +8,6 @@
       - /var/lib/ceph/bootstrap-osd/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-rgw/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-mds/{{ cluster }}.keyring
-      - /var/lib/ceph/bootstrap-rbd/{{ cluster }}.keyring
```

Comment:
why?

Reply:
to avoid a duplicate with https://github.com/ceph/ceph-ansible/pull/1988/commits/70e2787fe2970f4e255d013cd51658ebf3125ea3#diff-4d75e35ff4e0f2d65337ad52f08d8459L13

Reply:
right!